docs/primitives/a-collada-model.md

```diff
@@ -3,9 +3,10 @@ title: <a-collada-model>
 type: primitives
 layout: docs
 parent_section: primitives
-order: 5
 ---
 
+> COLLADA is not recommended for distributing over the web in production due to its large size. Check out using the [glTF component](https://github.com/xirvr/aframe-gltf) instead. You can convert from COLLADA to glTF using [the converter](https://cesiumjs.org/convertmodel.html).
```

Comment:
as I mentioned before, "due to its large size" is not correct; the COLLADA models are what's big. and link to the glTF page. I had a few other comments.